tests/new_unit_tests/Commands/Site/DeleteCommandTest.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace Pantheon\Terminus\UnitTests\Commands\Site;
+
+use Pantheon\Terminus\Commands\Site\DeleteCommand;
+use Pantheon\Terminus\UnitTests\Commands\CommandTestCase;
+
+/**
+ * Test suite for class for Pantheon\Terminus\Commands\Site\DeleteCommand
+ */
+class DeleteCommandTest extends CommandTestCase
+{
+
```

Comment:
Our PSR2-standard linting reacts to a blank line before the final braces in functions and classes, but not at the beginning.